dynawo/sources/Models/Modelica/Dynawo/Electrical/Loads/LoadAlphaBetaRestorative.mo

```diff
@@ -33,10 +33,19 @@ model LoadAlphaBetaRestorative "Generic model of a restorative Alpha-Beta load."
 
   equation
     if (running.value) then
-      tFilter * der(UFilteredRawPu) = ComplexMath.'abs' (terminal.V) - UFilteredRawPu;
+
       UFilteredPu = if UFilteredRawPu >= UMaxPu then UMaxPu elseif UFilteredRawPu <= UMinPu then UMinPu else UFilteredRawPu;
```

Comment:
maybe we can also put this one in the else (and put a UFilteredPu = 0 in the if). This way we keep the order of the equations

Reply:
done